rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/RMController.java

```diff
@@ -506,172 +510,150 @@ public void onFailure(Throwable caught) {
     }
 
     /**
-     * Parse the node state JSON string
-     * 
-     * @param json the "rm/monitoring" json result
-     * @return a POJO representation
+     * Model is rendered in the CompactView based on the new and old (current) model.
+     * That is why, we take old model and clone it. Then we process server response
+     * and alternate new model by adding/removing node/nodesources.
+     * @param json
      */
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace the type specification in this constructor call with the diamond operator ("<>"). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2293)